packages/ui/src/ui-component/InputParametersTooltip.js

```diff
@@ -0,0 +1,13 @@
+import { Info } from '@mui/icons-material';
+import { IconButton, Tooltip } from '@mui/material';
+import parse from 'html-react-parser';
+
+export const InputParameterTooltip = ({ title }) => {
```

Comment:
can we pull from latest master and replace all occurrence of `Tooltip` to use this instead? Going to decline #82 and have the bug #81 fixed as part of this PR